collectors/python.d.plugin/alarms/metadata.yaml

```
@@ -1,53 +1,152 @@
-    description: ''
-  keywords: []
+    description: ""
+  keywords:
+    - alarms
+    - netdata
   most_popular: false
 overview:
   data_collection:
-    metrics_description: ''
-    method_description: ''
+    metrics_description: |
+      This collector creates an 'Alarms' menu with one line plot of `alarms.status`.
+    method_description: |
+      Alarm status is read from the Netdata agent rest api [`/api/v1/alarms?all`](https://learn.netdata.cloud/api#/alerts/alerts1).
   supported_platforms:
     include: []
     exclude: []
   multi_instance: true
   additional_permissions:
-    description: ''
+    description: ""
   default_behavior:
     auto_detection:
-      description: ''
+      description: |
+        It discovers instances of Netdata running on localhost, and gathers metrics from `http://127.0.0.1:19999/api/v1/alarms?all`. `CLEAR` status is mapped to `0`, `WARNING` to `1` and `CRITICAL` to `2`. Also, by default all alarms produced will be monitored.
     limits:
-      description: ''
+      description: ""
     performance_impact:
-      description: ''
+      description: ""
 setup:
   prerequisites:
     list: []
   configuration:
     file:
-      name: ''
-      description: ''
+      name: python.d/alarms.conf
+      description: ""
     options:
-      description: ''
+      description: |
+        There are 2 sections:
+
+        * Global variables
+        * One or more JOBS that can define multiple different instances to monitor.
+
+        The following options can be defined globally: priority, penalty, autodetection_retry, update_every, but can also be defined per JOB to override the global values.
+
+        Additionally, the following table contains all the options that can be configured inside a JOB definition.
+
+        Every configuration JOB starts with a `job_name` value which will appear in the dashboard, unless a `name` parameter is specified.
       folding:
-        title: ''
+        title: Config options
         enabled: true
-      list: []
+      list:
+        - name: url
+          description: Netdata agent alarms endpoint to collect from. Can be local or remote so long as reachable by agent.
+          default_value: http://127.0.0.1:19999/api/v1/alarms?all
+          required: true
+        - name: status_map
+          description: Mapping of alarm status to integer number that will be the metric value collected.
+          default_value: '{"CLEAR": 0, "WARNING": 1, "CRITICAL": 2}'
+          required: true
+        - name: collect_alarm_values
+          description: set to true to include a chart with calculated alarm values over time.
+          default_value: false
+          required: true
+        - name: alarm_status_chart_type
+          description: define the type of chart for plotting status over time e.g. 'line' or 'stacked'.
+          default_value: "line"
+          required: true
+        - name: alarm_contains_words
+          description: |
```

When inside elements that are gonna be tables, use > to not mess with newlines
```suggestion
          description: >
```